### DISCUSSION TITLE
zwave_js adds three nodes instead of one

### DISCUSSION THREAD

**Discussion by austinmroczek** [zwave-js/zwave-js#6495] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

Reposting here vice in HA Core, per direction.  Possibly related to #6482 

I ran into a weird issue today after installing a device using Smart Start.

I installed a Zooz Zen52 LR dual relay (and nothing else).

In Smart Start I provisioned the DSK (manually typed) in Z-wavejsUI and called it “Front Patio”. I set it for S2 Authenticated only.

Then I energized the circuit. After a while Node 073 (pretty sure this was the next available node number) was found. The interview completed. Everything looked good, except it was showing as “no security”. Then less than a minute later the node was marked as dead. Then there are a handful of “Secure inclusion started”, then failed, then stopped.

Then node 077 is found and interviewed. It’s the same device. Also showing as “no security.” Then it is marked as dead almost immediately.

I then went on to do other things for a few hours. I came back and now there is a new node 090, showing S2 Authentication.

A few questions:
1) If I mark something as S2 Authentication only in Smart Start, why would it add a node with no security ?
2) If a node seems to fully add (even though it should have had security) and is then marked as dead, why would a new node be added?

Possibly related to https://github.com/zwave-js/node-zwave-js/pull/6483, though it does not seem to completely overlap this issue.

### Software versions

HA Core 2023.11.1
HA Supervisor 2023.10.1
HA Operating System 11.1
zwave-js-ui: 9.3.1
zwave-js: 12.2.3

### Device information

I have a Zooz ZST39 LR controller stick.

I installed a Zooz Zen52 LR dual relay (and nothing else).  It was given node ID 073, then 077, then 090.

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[z-ui_2023-11-04.log](https://github.com/zwave-js/node-zwave-js/files/13260469/z-ui_2023-11-04.log)
[zwavejs_2023-11-04.log](https://github.com/zwave-js/node-zwave-js/files/13260470/zwavejs_2023-11-04.log)

**Comment by AlCalzone** [comment, marked as answer]
> If I mark something as S2 Authentication only in Smart Start, why would it add a node with no security ?

It was attempted to include the node using S2 Authenticated, but this failed for some reason (usually bad connectivity). Using SmartStart, the including device then leaves the network, resets itself and tries again, which answers the next question:

> If a node seems to fully add (even though it should have had security) and is then marked as dead, why would a new node be added?

Your log does confirm this:
```
2023-11-04T17:56:29.713Z CNTRLR   [Node 073] Timed out while waiting for a response from the node (ZW0201)
2023-11-04T17:56:29.714Z CNTRLR   [Node 073] Security S2 bootstrapping failed: did not receive the node's desire
                                  d security classes.
2023-11-04T17:56:29.715Z CNTRLR   [Node 073] SmartStart inclusion failed. Checking if the node needs to be remov
                                  ed.
```

prior to this there were a few communication failures due to bad connectivity. Some commands are received at -99 dBm, which is fairly close to the background noise levels (even below for some of the channels):
```
channel 0: -109 dBm
channel 1: -86 dBm
channel 2: -86 dBm
```

https://zwave-js.github.io/node-zwave-js/#/troubleshooting/connectivity-issues?id=general-troubleshooting might help if you haven't already done this.
